scripts/make/test/behat.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# Standard build tools boilerplate
+script_args=$@
+script_path=$(dirname $0)
+working_dir=$(pwd)
+cd "$script_path"
+cd ../../..
+repo_root=$(pwd)
+
+source "$repo_root/.build.env"
+
+set -e
+
+# Install test dependencies on bitbucket pipelines.
+if [ $(command -v apk) ]; then
+  #apk update
+  #apk add build-base gcc abuild binutils libpng-dev autoconf automake build-base libtool nasm mysql-client
+  apk --no-cache add sqlite
+fi
+
+mv /usr/sbin/sendmail /usr/sbin/sendmail_back
+echo "exit 0" > /usr/sbin/sendmail
+chmod +x /usr/sbin/sendmail
+
+cd docroot
+../vendor/bin/drush si "$tests_behat_install_profile" --db-url=sqlite://../testdb.sqlite -y
```

Comment:
I feel like we should do mysql by default.

Reply:
Thanks @PeterBerryman 

It complicates the setup if we default to a MySQL container under Docker and rely on a MySQL service on Bitbucket Pipelines.

Can we go with sqlite for now and I'll do another PR down the road with an example pattern/setup if you want to use a MySQL container under Docker or MySQL service under Bitbucket Pipelines?